Parser/Client/Browser.php

```diff
@@ -68,7 +68,6 @@ class Browser extends AbstractClientParser
         'DW' => 'Dark Web Browser',
         '1B' => '115 Browser',
         '2B' => '2345 Browser',
-        '36' => '360 Phone Browser',
```

Comment:
wondering if it wouldn't be better to keep the record and maybe give it the same name as below, so the short code is still valid and usable.

Reply:
I returned the changes.